ONSdigital/dp-developer-site
# Fix/url encoding

### What

Implemented a `NewReplacer` to remove instances of parentheses in the `pathDir` strings. This is to provide cleaner URLs as parentheses were being encoded in the address bar.

### How to review

1. Pull latest changes.
2. Check endpoints with params, e.g., `/filters/{id}/dimensions/{name}` and ensure the address bar does not contain encodings for the parentheses.

### Who can review

Anyone
